### PR TITLE
Fixed Azure table property being not sanitized.

### PR DESCRIPTION
**Pull request by dVakulen**
Return values of pure method `string.Replace` weren't used.

### CODE REVIEW

File: src/OrleansAzureUtils/Storage/AzureStorageUtils.cs [hunk 1]
```diff
@@ -275,10 +275,10 @@ public static string SanitizeTableProperty(string key)
...
             // Remove any characters that can't be used in Azure PartitionKey or RowKey values
             // http://www.jamestharpe.com/web-development/azure-table-service-character-combinations-disallowed-in-partitionkey-rowkey/
-            key.Replace('/', '_');  // Forward slash
-            key.Replace('\\', '_'); // Backslash
-            key.Replace('#', '_');  // Pound sign
-            key.Replace('?', '_');  // Question mark
+            key = key.Replace('/', '_').  // Forward slash
+                Replace('\\', '_').       // Backslash
+                Replace('#', '_').        // Pound sign
+                Replace('?', '_');        // Question mark
```
Comment (jdom) [inline src/OrleansAzureUtils/Storage/AzureStorageUtils.cs]: I know I'm nit picking, but could you put the dots before the method invocation, such as `.Replace(`

Comment (jdom) [inline src/OrleansAzureUtils/Storage/AzureStorageUtils.cs]: BTW, feel free to push back on the comment if it's going to require a lot of context switching or fighting git from you, this is a VERY minor comment that I don't feel *absolutely must* be addressed :tongue:

**Comment by jdom** [comment]
Thanks for the PR. The code looks good... now :)

File: src/OrleansAzureUtils/Storage/AzureStorageUtils.cs (hunk 1 above)
Comment (dVakulen) [inline src/OrleansAzureUtils/Storage/AzureStorageUtils.cs]: Already fought and won.

**Comment by jdom** [comment]
Thanks @dVakulen for the quick turnaround :)

**Comment by sergeybykov** [comment]
:+1:

**Comment by gabikliot** [comment]
Great catch by @centur !

**Pull request merged**